Hw_02/Homework2.rmd

```diff
@@ -0,0 +1,157 @@
+acjminder <- gapminder %>% 
+  filter(country %in% c("Afghanistan","Canada","Japan") & year > 1969 & year < 1980)
+print(acjminder)
+```
+
+
+#1.2 Use the pipe operator to select "country" and "gdpPercap" from your filtered dataset in 1.1
+``` {r}
+acjminder %>% 
+  select(country, gdpPercap)
+```
+
+#1.3 Make a new variable in gapminder for the change in life expectancy from the previous measurement. Filter this table to show all of the entries that have experienced a drop in life expectancy. Hint: you might find the lag() and diff() functions useful.
+```{r}
+deltalifeExp <- gapminder %>% 
+  mutate(deltalifeExp = lag(lifeExp, order_by = year))
+arrange(deltalifeExp, year)
+deltalifeExp %>% 
+  filter(deltalifeExp < lifeExp)
+```
+
+#1.4 Filter gapminder so that it shows the max GDP per capita experienced by each country. Hint: you might find the max() function useful here.
+```{r}
+gapminder %>% 
+  group_by(country) %>% 
+  filter(gdpPercap %in% max(gdpPercap))
+```
+
+#1.5 Produce a scatterplot of Canada's life expectancy vs. GDP per capita using ggplot2, without defining a new variable. That is, after filtering the gapminder data set, pipe it directly into the ggplot function. In your plot, put GDP per capita on a log scale. 
+```{r}
+gapminder %>% 
+  filter(country == "Canada") %>% 
+  ggplot() +
+  aes(x = lifeExp,
+      y = gdpPercap) +
+  geom_point() + scale_y_log10()
+```
+
+#2.1 Pick two quantitative variables to explore. Make a table of descriptive statistics for these variables using summarize (include whatever statistics you feel appropriate). Make a scatterplot of these variables using ggplot()
+```{r}
+library(palmerpenguins)
+palmerpenguins::penguins
+
+penguin <- na.omit(penguins)
+penguin
+penguin %>% 
+  summarize(meanl = mean(bill_length_mm),
+            sdl = sd(bill_depth_mm),
+            meand = mean(bill_length_mm),
+            sdd = sd(bill_depth_mm),
+  )
+
+    ggplot(penguins) +
+      aes(x = bill_length_mm,
+          y = bill_depth_mm) +
+      geom_point()
+```
+
+#2.2 Pick one categorical variable and one quantitative variable to explore. Make a summary table giving the sample size (hint: n()) and descriptive statistics for the quantitative variable by group. Make one or more useful plots to visualize these variables.
+```{r}
+penguin %>% 
+  group_by(species) %>% 
+  summarize(meangd = mean(bill_depth_mm),
+            sdgd = sd(bill_depth_mm),
+            meangl = mean(bill_length_mm),
+            sdgl = sd(bill_length_mm),
+            meangf = mean(flipper_length_mm),
+            sdgf = sd(flipper_length_mm),
+            n = n())
+  ggplot(penguin) +
+      aes(x = bill_length_mm,
+          y = bill_depth_mm,
+          fill = species,
+          color = species) +
+      geom_point()
+  ggplot(penguin) +
+    aes(x = species) +
+   geom_bar()
+  ggplot(penguin) +
+  aes(x = bill_length_mm,
+      y = flipper_length_mm,
+      fill = species,
+      color = species) +
+  geom_point()
+
+```
+
+
```

Comment:
Is there something suppose to be after "geom_point()"? All I"m seeing is a long line of empty code.